src/events.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include "events_listener.h"
+#include "game_object.h" 
+#include "graphics_handler.h"
+#include "observable.h"
+#include <SDL2/SDL.h>
+
+class Events : public GameObject, public Observable<EventsListener> {
```

Comment:
Does an event need to be a game object? Can't events be triggered through other updates and don't need to be updated themselves.